packages/block-library/src/embed/save.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames/dedupe';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const { url, caption, type, providerNameSlug } = attributes;
+
+	if ( ! url ) {
+		return null;
+	}
+
+	const embedClassName = classnames( 'wp-block-embed', {
+		[ `is-type-${ type }` ]: type,
+		[ `is-provider-${ providerNameSlug }` ]: providerNameSlug,
+	} );
+
+	return (
+		<figure className={ embedClassName }>
```

Comment:
In `master`, if I paste a YouTube link and look at the markup of the generated embed, I see:

```html
<figure class="wp-block-embed-youtube wp-block-embed is-type-video is-provider-youtube wp-embed-aspect-16-9 wp-has-aspect-ratio">
```

The following classes seem to be missing in this branch:

* `wp-block-embed-${ providerNameSlug }`
* `wp-embed-aspect-16-9`
* `wp-has-aspect-ratio`

Reply:
> wp-embed-aspect-16-9
wp-has-aspect-ratio

These classes seem to work just fine. Can you share your testing link?

> wp-block-embed-${ providerNameSlug }

Previously this class was added before save as default block name class. With your previous comment for backwards compatibility I added it in `save` function. So it will not be visible before save but the final classes are identical (after save).

I believe this is okay, no?

Reply:
> These classes seem to work just fine. Can you share your testing link?

I don't see it now, I'm not sure how I tested. :) 👍 

> I believe this is okay, no?

Yep!